### PR TITLE
Jacoco

### PR DESCRIPTION
**Pull request by victornoel**
This is for artipie/npm-adapter#52.

I also cleaned a bit the pom in terms of dependencies and properties for easier integration with child projects.

### CODE REVIEW

**Comment by victornoel** [comment]
@g4s8 there doesn't seem to be any 0crat integration here so I'm pinging you

**Comment by victornoel** [comment]
@g4s8 can we merge this?

File: pom.xml [hunk 1]
```diff
@@ -72,10 +73,29 @@ SOFTWARE.
   </scm>
   <dependencyManagement>
     <dependencies>
+       <dependency>
+        <groupId>io.vertx</groupId>
+        <artifactId>vertx-dependencies</artifactId>
+        <version>${vertx.version}</version>
+        <type>pom</type>
```
Comment (g4s8) [inline pom.xml:80]: @victornoel why did you change this dependency?

File: pom.xml [hunk 2]
```diff
@@ -72,10 +73,29 @@ SOFTWARE.
...
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.junit</groupId>
+        <artifactId>junit-bom</artifactId>
+        <version>${junit-platform.version}</version>
+        <type>pom</type>
```
Comment (g4s8) [inline pom.xml:87]: @victornoel and this one?

File: pom.xml (hunk 1 above)
Comment (victornoel) [reply pom.xml:80]: @g4s8 I didn't change them, I changed how they were declared so we get all the dependencies management for all the vertx dependency at once instead of declaring them one by one. This is standard practice for handling this kind of dependencies, see https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies for details

File: pom.xml (hunk 2 above)
Comment (victornoel) [reply pom.xml:87]: @g4s8 same as above

**Comment by victornoel** [comment]
@g4s8 can we release a new version of the parent pom so that I can use it to solve artipie/npm-adapter#52?

**Comment by victornoel** [comment]
@g4s8 wait no, it seems I'm still missing something in this pom, I will make another PR, sorry

**Comment by victornoel** [comment]
@g4s8 it's in #21 :)
